src/sources/aws_s3/sqs.rs

```diff
@@ -379,12 +379,20 @@ impl IngestorProcess {
     }
 
     async fn handle_sqs_message(&mut self, message: Message) -> Result<(), ProcessingError> {
-        let s3_event: S3Event = serde_json::from_str(message.body.unwrap_or_default().as_ref())
+        let s3_event: Event = serde_json::from_str(message.body.unwrap_or_default().as_ref())
             .context(InvalidSqsMessageSnafu {
-                message_id: message.message_id.unwrap_or_else(|| "<empty>".to_owned()),
+                message_id: message.message_id.clone().unwrap_or_else(|| "<empty>".to_owned()),
             })?;
 
-        self.handle_s3_event(s3_event).await
+        match s3_event {
+            Event::TestEvent(_s3_test_event) => {
+                debug!(message = "Found S3 Test Event", ?message.message_id);
```

Comment:
```suggestion
                debug!(?message.message_id, message = "Found S3 Test Event.");
```

Passes the existing `check-events` scripts, and re-ordered to pass an upcoming change to the script.